docs/Installation.md

```diff
@@ -21,7 +21,7 @@
   * [5.4 Post-installation (FreeBSD)](#54-post-installation-freebsd)
 * [6. Installation on Ubuntu](#6-installation-on-ubuntu)
 * [7. Post-installation](#7-post-installation)
-  * [7.1 Sanity check](#71-sanity-check)
+  * [7.1 Check of Zonemaster::Backend](#71-check-of-zonemasterbackend)

```

Comment:
"Check" is a bit vague for a heading. "Sanity check" gives more context. "Smoke test" might be an alternative if you dislike the "sanity" word.
https://en.wikipedia.org/wiki/Sanity_check#Software_development
http://www.innovateus.net/innopedia/what-does-sanity-check-mean
https://en.wikipedia.org/wiki/Smoke_testing_(software)